src/witan/send/adroddiad/clerk/slides.clj

```diff
@@ -91,22 +91,28 @@
   (clerk/fragment
    (clerk/html
     {::clerk/width :full}
-    [:p.text-6xl.font-bold.font-sans (:title conf)])
+    [:div
+     [:p.text-6xl.font-bold.font-sans (:title conf)]
+     [:p.text-4xl.font-bold.font-sans (:sub-title conf)]])
    (mc-logo)))
 
 (defmethod slide ::was/title-body-slide [conf]
   (clerk/fragment
    (clerk/html
     {::clerk/width :full}
-    [:p.text-6xl.font-bold.font-sans (:title conf)])
+    [:div
+     [:p.text-6xl.font-bold.font-sans (:title conf)]
+     [:p.text-4xl.font-bold.font-sans (:sub-title conf)]])
    (box-type conf)
    (mc-logo)))
 
 (defmethod slide ::was/title-two-columns-slide [conf]
   (clerk/fragment
    (clerk/html
     {::clerk/width :full}
-    [:p.text-6xl.font-bold.font-sans (:title conf)])
+    [:div
+     [:p.text-6xl.font-bold.font-sans (:title conf)]
+     [:p.text-4xl.font-bold.font-sans (:sub-title conf)]])
```

Comment:
do we want to wrap up `[:p.text-4xl.font-bold.font-sans (:sub-title conf)]]` in an if like (when (:sub-title conf) [:p.text-4xl.font-bold.font-sans (:sub-title conf)]])` so we don't add in an empty `:p`?

Reply:
So I've tested that specifically and if it's empty it doesn't appear to create additional whitespace, so possibly no need to add an `if`